### PR TITLE
Ensure the output file was created

### PR DESCRIPTION
**Pull request by micalevisk**
I'm trying to use the new `output` input but didn't succeed (`No such file or directory` error). I'm proposing adding steps to the `main.yml` to ensure that the output file was created as expected

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -26,6 +26,8 @@ jobs:
           format: json
           output: /tmp/foo.json
           fail: true
+      - name: show permissions and number of lines of output json file
+        run: ls -l /tmp/foo.json && wc -l /tmp/foo.json
```
Comment (micalevisk) [inline .github/workflows/main.yml:30]: this is expected to work, right?

**Comment by mre** [comment]
Well it sorta works... it's just defined _inside_ the lychee-action itself and not outside.
Admittedly I didn't think this through and I couldn't think of a good use-case for it.

Guess [this](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#outputs) is the way to go. Declaring output parameters in an action's metadata file.
```yaml
outputs:
  output: # id of the output
    description: 'Path to lychee output file'
```

P.S.: Guess this shouldn't be a pull request but an issue. Pipeline tests can be made on a branch or a fork. 😉

**Comment by micalevisk** [comment]
oh right. I always miss this `outputs` field.

**Comment by micalevisk** [comment]
the problem with my workflow was using `/tmp`. Looks like GH drops the files on that directory across steps or something.

**Pull request closed without merging**